### DISCUSSION TITLE
Import data from excel, csv to variables

### DISCUSSION THREAD

**Discussion by deejayluk** [grafana/grafana#40558] [Data Sources]
Hi

Could you add posibility to import data from excel or csv to variables?

![image](https://user-images.githubusercontent.com/73607094/137695527-f8ce91c9-749d-44e9-8e82-6701f33047ab.png)

I have dashboard with variable name VALUE and if i paste some item name to this variable then show me data, but sometimes users send me few items name in excel file. Faster will be when I choose the file and import automatically than copying item by item.

This is only my suggestion :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
